### DISCUSSION TITLE
Is there a convar for SBoxLimit_gmod_contr_spawners ?

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [wiremod/advdupe2#349] [Q&A] [answered]
When trying to spawn a dupe that uses adv-dupe2 / wire2 'wire contraption spawner' I am getting the sandbox limit error with the 'broken' placeholder text "Error: SBoxLimit_gmod_contr_spawners" as far as I know there is no convar to set the max number of contraption spawners allowed. Tried to look in server and wiremod/advdupe config to find out, tried to guess the convar.
**If there is in fact a command for setting the max number of contraption spawners, what is it?**

**Comment by thegrb93** [comment, marked as answer]
sbox_maxgmod_contr_spawners or sbox_maxcontr_spawners
